rero_ils/modules/holdings/tasks.py

```diff
@@ -36,13 +36,19 @@ def delete_standard_holdings_having_no_items():
     errors = 0
     for hit in [hit for hit in es_query.scan()]:
         record = Holding.get_record(hit.meta.id)
-        try:
-            record.delete(force=False, dbcommit=True, delindex=True)
-        except Exception as err:
-            errors += 1
-            reasons = record.reasons_not_to_delete()
-            current_app.logger.error(
-                f'Can not delete standard holding: {hit.pid} {reasons} {err}')
+        if record:
+            try:
+                record.delete(force=False, dbcommit=True, delindex=True)
+            except Exception as err:
+                errors += 1
+                reasons = record.reasons_not_to_delete()
+                current_app.logger.error(
+                    f'Can not delete standard holding: '
+                    f'{hit.pid} {reasons} {err}')
+        else:
+            # delete holding from index
+            HoldingsIndexer().client.delete(
+                id=hit.meta.id, index='holdings', doc_type='_doc')
 
     counts = {'count': es_query.count(), 'errors': errors}
```

Comment:
May be the count is not correct here. Is this the remaining count or the count we tried to delete?

Reply:
I've rewrite that